src/server/enrollment-system.js

```diff
@@ -241,19 +241,19 @@ function veteranToSpouseInfo(veteran) {
  */
 function resourceToIncomeCollection(resource) {
   const incomeCollection = [];
-  if (resource.grossIncome) {
+  if (resource.grossIncome !== undefined) {
```

Comment:
This is going to allow nulls through, which might trip us up later. Maybe `!isNaN(resource.grossIncome)`?